autogpt_platform/frontend/src/stories/Typography.stories.tsx

```diff
@@ -0,0 +1,177 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Text } from "@/components/_new/Text/Text";
+import { StoryCode } from "@/stories/helpers/StoryCode";
+
+const meta: Meta<typeof Text> = {
+  title: "Design System/ Tokens /Typography",
+  component: Text,
+  parameters: {
+    layout: "fullscreen",
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof Text>;
```

Comment:
This does not appear to be used

Reply:
I plan to add an ESLint/VScode config so that unused vars are removed when saving a file 😪 🙏🏽